### ISSUE TITLE
make a provision to allow user to enter password on prompt alongside the command line

### ISSUE THREAD

**Issue by sudipbhandari126** [softwarescales/git-issues#18]
I hope it's a minor thing but it would be convenient for user to be able to enter password on prompt rather than mandatorily having to mention in command line always. Like in Mysql or  Linux.

**Comment by IonicaBizau** [comment]
@sudipbhandari126 True. I'm not sure if it's good idea or not, but such a thing could be implemented in the [`tilda`](https://github.com/IonicaBizau/tilda) core (https://github.com/IonicaBizau/tilda/issues/4)

For now you can simply save the username and the password (can be a simple access token in `~/.git-issues`). 🚀

**Comment by IonicaBizau** [comment]
That's now possible with `tilda`. Works pretty much like the MySQL client. If you don't pass the password, it will ask you about it.

We have to switch from `clp` to `tilda`. Contributions are welcome! :four_leaf_clover: 
If not, I'll do it, but not in the near future.
